controllers/user_auth.go

```diff
@@ -18,6 +18,7 @@ func (ctrl UserAuthController) Routes(r *gin.RouterGroup) {
 	r.GET("/login/", ctrl.Login)
 	r.POST("/login/", ctrl.BasicLogin)
 	r.POST("/identify/", ctrl.Identify)
+	r.POST("/code/", ctrl.Recover)
```

Comment:
The endpoint URL should be more intuitive.

Reply:
I think it is ok as Facebook also uses this for OTP submission page.